### DISCUSSION TITLE
upbge installer

### DISCUSSION THREAD

**Discussion by Reaper10** [UPBGE/upbge#1460] [Ideas/Features for UPBGE 0.3+]
Will ever be a installer for upbge?
If so I would rename the exe to upbge. This will cut down on confusion.

**Comment by youle31** [comment]
As it is not a bug, I move this thread to discussions.

**Comment by lordloki** [comment]
At this moment, it is not a priority to setup an installer.

By other way, the rename to upbge.exe is in our TODO before to release the 0.3.

**Reply by buddha314** [reply]
what is the status of the .exe?
